stacks/SiriGeneratorStack.ts

```diff
@@ -11,10 +11,13 @@ export function SiriGeneratorStack({ stack }: StackContext) {
         name: `cdd-siri-sx-${stack.stage}`,
```

Comment:
It's also worth enabling versioning on the validated siri bucket

Reply:
done